### DISCUSSION TITLE
画廊瀑布流

### DISCUSSION THREAD

**Discussion by YJiva** [lsky-org/lsky-pro#620] [灌水]
想把画廊改成瀑布流的时候突然发现已经写好了初始化，只要将同时有w-full h-32这两个class属性在的那个div把这两个class去掉就瀑布了，好酷

**Comment by lakelandc** [comment]
你好，想請問這個具體是怎麼做到的？我用的是企業版，images 中似乎沒看到你說的這兩個 class。

**Reply by shijiew** [reply]
企业版可以公开展示图片吗，开源版的好像不行

**Reply by YJiva** [reply]
大概是在source code的resources/views/common/gallery.blade.php里
